roles/kibana/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Install Kibana.
+  package:
+    name: kibana
+    state: present
+
+- name: Copy Kibana configuration.
+  template:
+    src: kibana.yml.j2
```

Comment:
Nit: We don't really have .j2 on our templates. Also, we keep them in templates/ with similar path that they'll be deployed to.